client/src/components/DataFiles/DataFilesModals/DataFilesMoveModal.jsx

```diff
@@ -49,6 +49,7 @@ const DataFilesMoveModal = React.memo(() => {
       api: 'tapis',
       scheme: 'private',
       system: systems[0].system,
+      path: `${systems[0].homeDir || ''}`,
```

Comment:
here or in the other PP (#765) can we update `client/src/components/DataFiles/fixtures/DataFiles.systems.fixture.js`?

Reply:
I do think updating the fixtures should be a separate task that should fall under the [Files Unit Test](https://jira.tacc.utexas.edu/browse/TV3-101) task because changes here will impact a lot of other tests as well. Let me know what you think @nathanfranklin @rstijerina

Reply:
Cool. sounds good to me. I see these tests running with an updated fixture and then all the unrelated tests failing.